ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -389,11 +389,21 @@ export const PageProviderDisplayBehavior = [
     },
 
     _isSelected(item) {
-      return !!(this.selectedItems && this.selectedItems.length && this.selectedItems.indexOf(item) > -1);
+      return this.multiSelection
+        ? !!(this.selectedItems && this.selectedItems.length && this.selectedItems.indexOf(item) > -1)
+        : !!(this.selectedItem && this.selectedItem === item);
     },
 
     _isIndexSelected(index) {
-      return !!(this.selectedItems && this.selectedItems.length && this.selectedItems.indexOf(this.items[index]) > -1);
+      return this.multiSelection
+        ? !!(
+            this.selectedItems &&
+            this.selectedItems.length &&
+            this.items &&
+            this.items.length > index &&
+            this.selectedItems.indexOf(this.items[index]) > -1
+          )
+        : !!(this.selectedItem && this.items && this.items.length > index && this.selectedItem === this.items[index]);
```

Comment:
Should we add some unit tests for this?

Reply:
Maybe it's a good idea 👍  Let me see how we can improve it.